### PR TITLE
[MM-14375] Fix flaky E2E tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix flaky E2E tests

Not critical for v5.10 release but labelled since it's scheduled for it :smile:.

#### Ticket Link
Jira: [MM-14375](https://mattermost.atlassian.net/browse/MM-14375)

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -176,11 +173,11 @@ Cypress.Commands.add('postMessageReplyInRHS', (message) => {
 });
 
 Cypress.Commands.add('getLastPost', () => {
-    return cy.get('#postListContent').children().last();
+    return cy.get('#postListContent').children().children().children().children().children().first();
 });
 
 Cypress.Commands.add('getLastPostId', () => {
-    return cy.get('#postListContent').children().last().invoke('attr', 'id').then((divPostId) => {
+    return cy.get('#postListContent').children().children().children().children().children().first().invoke('attr', 'id').then((divPostId) => {
```
Comment (saturninoabril) [inline cypress/support/commands.js]: Deeply nested access to children of postListContent can't be avoid due to the external components (`AutoSizer `, `DynamicSizeList `) introduced recently by [virtualized post list](https://github.com/mattermost/mattermost-webapp/blob/ef76bf9161e665792cb77277ff1fbee9c7dd1aeb/components/post_view/post_list.jsx#L465).

Comment (cometkim) [reply cypress/support/commands.js]:
```suggestion
    return cy.get('#postListContent [id^=post]:first').invoke('attr', 'id').then((divPostId) => {
```

Comment (saturninoabril) [reply cypress/support/commands.js]: That's sweet, looks good to me, thanks @cometkim!

**Comment by saturninoabril** [comment]
@cometkim ready for another review, thanks

**Comment by cometkim** [comment]
Changes LGTM :)

**Pull request merged**